### ISSUE TITLE
Support freezing certain layers including bn

### ISSUE THREAD

**Issue by zhanghanduo** [Megvii-BaseDetection/YOLOX#121]
I have to manually add some "requires_grad" options and consider batchnorm when I want to freeze some certain layers. Maybe you could add some flags to enable this. thanks!

**Comment by FateScript** [comment]
Sorry for not replying in time. You could override get_model and define layers you want to freeze. Freezing certain layers is not very common. Feel free to reopen if you have any further quesions.
